srcpkgs/cura-fdm-materials/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'cura-fdm-materials'
 pkgname=cura-fdm-materials
-version=3.4.1
+version=3.6.0
 revision=1
 noarch=yes
-build_style=cmake
 wrksrc="fdm_materials-${version}"
+build_style=cmake
```

Comment:
why did this line get moved down

Reply:
xlint complained about it

Reply:
https://travis-ci.org/void-linux/void-packages/jobs/456307671